### ISSUE TITLE
Malicious code injection opportunity

### ISSUE THREAD

**Issue by crism** [anvc/scalar#64]
When loading media resources from a remote service, the XSL transformation file is specified in the form submission parameters as a URI. The Scalar server fetches this resource from the specified URI and applies it; the form submission could easily be presented with a maliciously-crafted XSLT, and Scalar will naïvely execute it. The remote-resource processing XSLT should only be loaded from the local filesystem, installed with Scalar or with a trusted plugin.

**Comment by craigdietrich** [comment]
Yes, great catch!  I've been meaning to validate the values against Scalar's list of archives (archives.rdf).  Will add to the list.

**Comment by craigdietrich** [comment]
@crism Would you be able to put this on our repository as a pull request?

**Comment by craigdietrich** [comment]
Has this been included into anvc/scalar's repository?

**Comment by crism** [comment]
#62 is still open, and the PR #66 is still open as well; accepting #66 will fix this. I didn’t make a PR specifically to fix this (#64), but I could try and tease it apart if you want to fix this without accepting #66.
